libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/logger/SalesforceLogger.java

```diff
@@ -52,6 +53,11 @@
  */
 public class SalesforceLogger {
 
+    /**
+     * The factory for Salesforce log receivers.  Defaults to null
+     */
+    public static SalesforceLogReceiverFactory logReceiverFactory;
```

Comment:
Should it be a public static member or should we have a static method to provide a factory (and keep the member private)?